src/network/gun/version.2/hash.ts

```diff
@@ -22,26 +22,30 @@ export const hashPostSalt = memoizePromise(async function(postSalt: string) {
 
     const hash = (await Gun.SEA.work(postSalt, hashPair))!
     return hash.substring(0, N)
-})
+}, undefined)
 
 /**
  * @param key - The key need to be hashed
- * @param stableHash - Set this to true if you're writing new code.
- * Unstable hash may cause problem but we cannot just switch to stable hash
- * because it may breaks current data.
+ * @deprecated
  */
-export const hashCryptoKey = memoizePromise(async function(key: CryptoKey, stableHash: boolean) {
+export const hashCryptoKeyUnstable = memoizePromise(async function(key: CryptoKey) {
     const hashPair = `10198a2f-205f-45a6-9987-3488c80113d0`
     const N = 2
 
-    if (stableHash === true) {
-        const jwk = await CryptoKeyToJsonWebKey(key)
-        if (!jwk.x || !jwk.y) throw new Error('Invalid key')
-        const hash = (await Gun.SEA.work(jwk.x! + jwk.y!, hashPair))!
-        return hash.substring(0, N)
-    } else {
-        const jwk = JSON.stringify(await CryptoKeyToJsonWebKey(key))
-        const hash = (await Gun.SEA.work(jwk, hashPair))!
-        return hash.substring(0, N)
-    }
-})
+    const jwk = JSON.stringify(await CryptoKeyToJsonWebKey(key))
```

Comment:
Someone should `console.log` the jwk and grab an ordering of the keys for future historians. It should be deterministic for each crypto.subtle implementation at least, since the ordering for string keys depends only on order of assignment.

Reply:
Actually `hashCryptoKeyUnstable` is deprecated and we should use `hashCryptoKey` for any new code.

Reply:
it will be good trivia.